go.mod1

```diff
@@ -1,38 +1,209 @@
 module github.com/pingcap/tidb-tools
 
 require (
-	github.com/BurntSushi/toml v1.1.0
+	github.com/BurntSushi/toml v1.2.0
 	github.com/DATA-DOG/go-sqlmock v1.5.0
-	github.com/Shopify/sarama v1.31.1
 	github.com/coreos/go-semver v0.3.0
 	github.com/go-sql-driver/mysql v1.6.0
-	github.com/golang/protobuf v1.5.2
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/pingcap/check v0.0.0-20211026125417-57bd13f7b5f0
 	github.com/pingcap/errors v0.11.5-0.20211224045212-9687c2b0f87c
 	github.com/pingcap/failpoint v0.0.0-20220423142525-ae43b7f4e5c3
-	github.com/pingcap/kvproto v0.0.0-20220517085838-12e2f5a9d167
+	github.com/pingcap/kvproto v0.0.0-20220804022843-f006036b1277
 	github.com/pingcap/log v1.1.0
-	github.com/pingcap/tidb v1.1.0-beta.0.20220617092035-96526518b4e8
-	github.com/pingcap/tidb/parser v0.0.0-20220617085035-580a68dff3a8
-	github.com/pingcap/tiflow v0.0.0-20220622093837-49960d68d8b7
-	github.com/pingcap/tipb v0.0.0-20220602075447-4847c5d68e73
+	github.com/pingcap/tidb v1.1.0-beta.0.20220819091852-41b9e26d1ef4
+	github.com/pingcap/tidb/parser v0.0.0-20220819091852-41b9e26d1ef4
+	github.com/pingcap/tiflow v0.0.0-20220905041455-ca31feec87f5
 	github.com/siddontang/go v0.0.0-20180604090527-bdc77568d726
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.7.2-0.20220504104629-106ec21d14df
-	github.com/tikv/pd/client v0.0.0-20220307081149-841fa61e9710
-	go.etcd.io/etcd/api/v3 v3.5.2
+	github.com/stretchr/testify v1.8.0
+	github.com/tikv/pd/client v0.0.0-20220725055910-7187a7ab72db
 	go.etcd.io/etcd/client/v3 v3.5.2
 	go.etcd.io/etcd/tests/v3 v3.5.2
 	go.uber.org/atomic v1.9.0
 	go.uber.org/zap v1.21.0
-	golang.org/x/net v0.0.0-20220516155154-20f960328961
-	google.golang.org/grpc v1.46.2
+	google.golang.org/grpc v1.48.0
+)
+
+require (
```

Comment:
🤔

Reply:
It seems that newer version of go mod split indirect and direct pkg.